### ISSUE TITLE
Changes in the ENDESA web page.  login fail

### ISSUE THREAD

**Issue by nachoplus** [trocotronic/edistribucion#15]
Hello,

Something has change in the login page. Now it fails to connect:

requests.exceptions.ConnectionError: HTTPSConnectionPool(host='zonaprivada.edistribucion.comhttps', port=443): Max retries exceeded with url: //static.lightning.force.com/eu25/auraFW/javascript/Vo_clYDmAijdWOzW3-3Mow/aura_prod.js (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f48225ec160>: Failed to establish a new connection: [Errno -2] Name or service not known'))

If I try to visit with a web browser https://static.lightning.force.com/ I got this error:

Service Unavailable - DNS failure
The server is temporarily unable to service your request. Please try again later.
Reference #11.de497b5c.1675415417.3a97c1

Really I dont know how to help to fix this error so I reported to you 

Thanks

**Comment by Arman103** [comment]
Hi nachoplus,

I had the same error and after investigating it I see that something changed in the web portal, modifiying the code in function __force_login(self, recursive=False) ("Edistribucion.py") and moving the sentence "r = self.__get_url(upr.scheme+'://'+upr.netloc+src)" inside the below if it works again. To summarize:

NOT WORKING CODE:

            r = self.__get_url(upr.scheme+'://'+upr.netloc+src)

            if ('resources.js' in src):
                unq = unquote(src)
                self.__context = unq[unq.find('{'):unq.rindex('}')+1]
                self.__appInfo = json.loads(self.__context)
        logging.info('Performing login routine')

WORKING FINE AGAIN CODE:

REMOVE OR COMMENT THIS LINE:   r = self.__get_url(upr.scheme+'://'+upr.netloc+src) 

            if ('resources.js' in src):
                r = self.__get_url(upr.scheme+'://'+upr.netloc+src)
                unq = unquote(src)
                self.__context = unq[unq.find('{'):unq.rindex('}')+1]
                self.__appInfo = json.loads(self.__context)

Hope this helps.

Regards.

**Comment by nachoplus** [comment]
It wotks again!

Thank you!!

**Comment by trocotronic** [comment]
Can you make a PR?

**Comment by Arman103** [comment]
Done: [(https://github.com/trocotronic/edistribucion/pull/17) ]

**Comment by nachoplus** [comment]
I sent also a PR with the same content. **Please discard.**  I read wrongly the msg in my mobile and I thought that @trocotronic was asking to me to do. Sorry,

**Comment by trocotronic** [comment]
Fixed in #17

**Comment by nikuxic** [comment]
Hi everyone! Just to give more info...
I has same issue as you all. On Friday (when this 'hotfix' was published) I tried the new code and it was not working on my side either.
Yesterday I realised that old code was working again; probably webpage has changed again. So, on my side, both new and old code are now working.
Last, I just want to notice that version number of the code has not been increased.
